spec/trimmerdecorator_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../trimmerdecorator'
+
+describe TrimmerDecorator do
+  before :each do
+    @trimmer_decorator = TrimmerDecorator.new('Ichsan')
+  end
+
+  it 'should return an instance of TrimmerDecorator' do
+    expect(@trimmer_decorator).to be_an_instance_of TrimmerDecorator
+  end
```

Comment:
- [ ] Same remark, write a test to show that the trimmer_decorator works, for example, if the name is `Jonathanuous` it should trimmer and correct the name by using the `correct_name` method and supposed to be eq `Jonathanuo` 🙃 🙃